lib/inspec/cli.rb

```diff
@@ -330,6 +323,7 @@ def detect
   option :distinct_exit, type: :boolean, default: true,
     desc: "Exit with code 100 if any tests fail, and 101 if any are skipped but none failed (default).  If disabled, exit 0 on skips and 1 for failures."
   def shell_func
+    require "inspec/resources" # TODO: remove me if possible!
```

Comment:
Since the shell is the REPL, if we were multi-threaded, we could be sneaky and load fast to get to a prompt, then do our heavy loading while waiting on the user to type... if threading was safe on RSpec...